application/dash_application/views/pre_exome.py

```diff
@@ -191,7 +191,9 @@ def generateDebugLine(click, runs, firstsort, secondsort, colourby,
 layout = html.Div(className='body',
     children=[
         navbar("Pre-Exome"),
-        html.Div(className='sidebar',
+        html.Div(className='row flex-container',
```

Comment:
Could there be an explanation (or link to explanation) of these class names somewhere?

Reply:
Honestly, I copied it from the template. I'm not entirely sure what these really do.